content/en/blog/_posts/2023-04-11-volume-group-snapshot-alpha.md

```diff
@@ -0,0 +1,165 @@
+
+## As a storage vendor, how do I add support for group snapshots to my CSI driver?
+To implement the volume group snapshot feature, a CSI driver MUST:
+* Implement a new group controller service.
+* Implement group controller RPCs: `CreateVolumeGroupSnapshot`, `DeleteVolumeGroupSnapshot`, and `GetVolumeGroupSnapshot`.
+* Add group controller capability `CREATE_DELETE_GET_VOLUME_GROUP_SNAPSHOT`.
+
+See the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md) and the [Kubernetes-CSI Driver Developer Guide](https://kubernetes-csi.github.io/docs/) for more details.
+
+Although Kubernetes poses as little prescriptive on the packaging and deployment of a CSI Volume Driver as possible, it provides a suggested mechanism to deploy a containerized CSI driver to simplify the process.
+
+As part of this recommended deployment process, the Kubernetes team provides a number of sidecar (helper) containers, including the [external-snapshotter sidecar container](https://kubernetes-csi.github.io/docs/external-snapshotter.html) which has been updated to support volume group snapshot.
+
+The external-snapshotter watches the Kubernetes API server for the `VolumeGroupSnapshotContent` object and triggers `CreateVolumeGroupSnapshot` and `DeleteVolumeGroupSnapshot` operations against a CSI endpoint.
+
+## What are the limitations?
+The alpha implementation of volume group snapshots for Kubernetes has the following limitations:
+* Restore is achieved by creating a new PVC from a VolumeSnapshot that is part of a VolumeGroupSnapshot.
+* Does not support reverting an existing PVC to an earlier state represented by a snapshot that is part of a group snapshot (only supports provisioning a new volume from a snapshot).
+* No application consistency guarantees beyond any guarantees provided by the storage system (e.g. crash consistency).
+
+## What’s next?
+Depending on feedback and adoption, the Kubernetes team plans to push the CSI Group Snapshot implementation to Beta in either 1.28 or 1.29. Some of the features we are interested in supporting include volume replication, replication group, volume placement, application quiescing, changed block tracking, and more.
+
+## How can I learn more?
+The code repository for volume group snapshot APIs and controller is here: https://github.com/kubernetes-csi/external-snapshotter
+
+Check out additional documentation on the group snapshot feature here: https://kubernetes-csi.github.io/docs/
+
+## How do I get involved?
+This project, like all of Kubernetes, is the result of hard work by many contributors from diverse backgrounds working together. We offer a huge thank you to the contributors who stepped up these last few quarters to help the project reach alpha:
```

Comment:
```suggestion
This project, like all of Kubernetes, is the result of hard work by many contributors from diverse backgrounds working together. On behalf of SIG Storage, I would like to offer a huge thank you to the contributors who stepped up these last few quarters to help the project reach alpha:
```

“We” does not make sense when there is only one author and without providing more context.

Reply:
Done